corehq/apps/receiverwrapper/tests/test_audit_logging.py

```diff
@@ -37,38 +38,59 @@ def setUpClass(cls):
         )
-        web_user = WebUser.create(self.domain, self.username, self.password,
-                                  None, None, role_id=role.get_id)
-        self.addCleanup(web_user.delete, None, None)
-        return web_user
+        user = user_cls.create(self.domain, self.username, self.password,
+                               None, None, role_id=role.get_id)
+        self.addCleanup(user.delete, None, None)
+        return user
 
     def assert_api_response(self, status, url):
         self.client.login(username=self.username, password=self.password)
         self.assertEqual(self.client.post(url).status_code, status)
 
-    def test_valid_request_not_logged(self, mock_record_metrics, mock_couchform_instance):
+    def test_api_user_api_endpoint(self):
         url = reverse(post_api, args=[self.domain])
-        self._create_user(edit_data=True, access_api=True)
+        self._create_user(access_api=True, access_mobile_endpoints=False)
         with patch('corehq.apps.receiverwrapper.views.notify_exception') as mock_notify_exception:
             self.assert_api_response(200, url)
             mock_notify_exception.assert_not_called()
 
-    def test_invalid_request_logged(self, mock_record_metrics, mock_couchform_instance):
+    def test_web_user_regular_submission(self):
         url = reverse(secure_post, args=[self.domain])
-        user = self._create_user(edit_data=True, access_api=True)
+        self._create_user(access_api=False, access_mobile_endpoints=True)
+        with patch('corehq.apps.receiverwrapper.views.notify_exception') as mock_notify_exception:
+            self.assert_api_response(200, url)
+            mock_notify_exception.assert_not_called()
+
+    def test_commcare_user_regular_submission(self):
+        url = reverse(secure_post, args=[self.domain])
+        self._create_user(access_api=False, access_mobile_endpoints=True, user_cls=CommCareUser)
+        with patch('corehq.apps.receiverwrapper.views.notify_exception') as mock_notify_exception:
+            self.assert_api_response(200, url)
+            mock_notify_exception.assert_not_called()
+
+    def test_api_user_regular_submission_gets_logged(self):
+        url = reverse(secure_post, args=[self.domain])
+        user = self._create_user(access_api=True, access_mobile_endpoints=False)
         with patch('corehq.apps.receiverwrapper.views.notify_exception') as mock_notify_exception:
             self.assert_api_response(200, url)
             mock_notify_exception.assert_called_once_with(ANY, message=ANY)
             args, kwargs = mock_notify_exception.call_args
 
             self.assertIsInstance(args[0], WSGIRequest)
 
-            expected_message = (f'Restricted access by user samiam with id {user.get_id} and app_id None. '
-                                f'Error details: Request not made from post_api handler for user')
+            expected_message = f"NoMobileEndpointsAccess: invalid request by {user.get_id} on {self.domain}"
```

Comment:
for what it's worth, I put this `NoMobileEndpointsAccess` string in there so it'd be easy to find this error in sentry